tools/benchmark/src/mocha/runner.ts

```diff
@@ -33,16 +54,63 @@ import { runBenchmark } from "./runBenchmark";
  * @public
  */
 export function benchmark(args: BenchmarkArguments): Test {
-	const benchmarkTypeTag = BenchmarkType[args.type ?? BenchmarkType.Measurement];
-	const testTypeTag = TestType[TestType.ExecutionTime];
-	let qualifiedTitle = `${performanceTestSuiteTag} @${benchmarkTypeTag} @${testTypeTag} ${args.title}`;
+	return supportParentProcess({
+		title: qualifiedTitle(args),
+		only: args.only,
+		run: async () => {
+			// Create and run a benchmark if we are in perfMode, else run the passed in function normally
+			if (isInPerformanceTestingMode) {
+				const stats = await runBenchmark(args);
+				return stats;
+			} else {
+				const { benchmarkFn: argsBenchmarkFn } = validateBenchmarkArguments(args);
+				await args.before?.();
+				await argsBenchmarkFn();
+				args.beforeEachBatch?.();
+				await args.after?.();
+				await Promise.resolve();
+				return { error: "Not running in performance testing mode, so no data collected" };
```

Comment:
Does this surface in any way in the output when running the tests without performance mode?

Reply:
It shouldn't, unless you use the reporter.